cql3/expr/expression.hh

```diff
@@ -379,6 +480,15 @@ extern expression replace_column_def(const expression&, const column_definition*
 // For example this changes token(p1, p2) < token(1, 2) to my_column_name < token(1, 2).
 extern expression replace_token(const expression&, const column_definition*);
 
+
+extern ::shared_ptr<term> prepare_term(const expression& expr, database& db, const sstring& keyspace, const column_specification_or_tuple& receiver);
+
+extern assignment_testable::test_result test_assignment(const expression& expr, database& db, const sstring& keyspace, const column_specification& receiver);
+
+extern assignment_testable::test_result test_assignment_all(const std::vector<expression>& exprs, database& db, const sstring& keyspace, const column_specification& receiver);
+
+extern shared_ptr<assignment_testable> as_assignment_testable(expression e);
```

Comment:
If there were blurbs on the method-versions of these, let's please preserve them.  If not, this is fine.

Reply:
Will check and update.

Reply:
Done